_postinstall.js

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env node
```

Comment:
Can we revert that change? I believe it's not necessary

Reply:
yes, I agree, it is not necessary to delete it

Reply:
done!